src/main/kotlin/blackjack/controller/Controller.kt

```diff
@@ -0,0 +1,116 @@
+package blackjack.controller
+
+import blackjack.model.Deck
+import blackjack.model.FinalResult
+import blackjack.model.GamblerInfo
+import blackjack.model.Player
+import blackjack.view.InputView
+import blackjack.view.OutputView
+
+class Controller() {
+    private var players: List<Player> = mutableListOf()
+    private val dealer: Player = Player(GamblerInfo("dealer"))
+    private val deck = Deck()
+
+    fun run() {
+        try {
+            createPlayers()
+            OutputView.displayNamesOfPlayers(players)
+            roundOne()
+            OutputView.displayCardsOfDealer(dealer)
+            players.forEach { OutputView.displayCardsOfPlayers(it) }
+            OutputView.printEmptyLine()
+            players.forEach { playerTakesTurn(it) }
+            dealerTakesTurn()
+            OutputView.printEmptyLine()
+            OutputView.displayCardsOfPlayersWithScore(dealer)
+            players.forEach { OutputView.displayCardsOfPlayersWithScore(it) }
+            OutputView.printEmptyLine()
+            OutputView.displayFinalResultsHeading()
```

Comment:
I see consecutive use of `OutputView`.
Why don't we make it so that one `OutputView` method call covers it?
I think `printEmptyLine` can be hidden in individual methods in `OutputView`.